### ISSUE TITLE
Works on Raspberry Pi 2 with OpenElec and Kodi 14.1 and 14.2b

### ISSUE THREAD

**Issue by LaundroMat** [cees-elzinga/script.xbmc.hue.ambilight#48]
Hi -- 

I just wanted to mention that the script works fine (I had to manually set the bridge IP and create a user through the Hue API myself though) on my Raspberry PI 2 running OpenElec (Kodi 14.1 and 14.2 beta). I'm using 2 Philips Bloom Hue lights.

**Comment by dkruyt** [comment]
Yes, works also here with RPi2 and Ambilight mode... Maybe change the readmethat RPi2 is supported...
